### PR TITLE
ui update release 1: modals, landing, navlinks (more rolling releases to...

### PR DESCRIPTION
**Pull request by rebekahkwon**
... come)

### CODE REVIEW

File: client/scripts/components/layouts/landing.jsx [hunk 1]
```diff
@@ -22,56 +21,61 @@ var Landing = React.createClass({
...
-        <div>
-          <div id="tenWidth"></div>
-          <div id="twentyfiveWidth">
-            <img src="../../../images/Safety.png" />
-            <div>
-              <h4>Secure</h4>
-              <p>encryption technology in the voting industry</p>
+        <div className="content">
+          <div>
+            <div id="tenWidth"></div>
```
Comment (ryyann) [inline client/scripts/components/layouts/landing.jsx]: You can't use ID's multiple times, please use classes for this.

File: client/styles/_landing-page.scss [hunk 2]
```diff
@@ -1,13 +1,85 @@
...
+  height: 160px;
+  background-image: url('../../images/finiteelement_white.png');
+  background-size: 104px;
+  background-repeat: no-repeat;
+  background-position: center;
+  opacity: 0.6;
+}
+
+.voteblockbg {
+  background: -webkit-linear-gradient(left, #367CCF , #4AADA5);
```
Comment (ryyann) [inline client/styles/_landing-page.scss]: Auto-prefixer will take care of browser-prefixing, don't worry about adding them yourself :)

**Comment by ryyann** [comment]
Just need the above fixes, but otherwise things look good. We should go over some SASS stuff that will save you time writing some of this though.

File: client/scripts/components/layouts/landing.jsx (hunk 1 above)
Comment (rebekahkwon) [inline client/scripts/components/layouts/landing.jsx]: this was legacy code, i'll clean it up.

File: client/styles/_landing-page.scss (hunk 2 above)
Comment (rebekahkwon) [inline client/styles/_landing-page.scss]: cool, would keeping them introduce bugs now?

**Comment by ryyann** [comment]
Wheeeeeee!

**Pull request merged**